components/eamxx/src/diagnostics/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ set(DIAGNOSTIC_SRCS
   aodvis.cpp
   number_path.cpp
   aerocom_cld.cpp
+  atm_tend.cpp
```

Comment:
how about "AtmTimeDeriv"? It is a FD approx of the time deriv, after all. The name "diff" suggests no scaling by 1/dt, so I think that would also be misleading...

Reply:
In EAMxx, the word "tendency" means very little, since atm procs simply update the desired fields. Even in EAM, if I understand it correctly, tendency means what you are doing here, since phys packages are simply doing (if I understand correctly):
- compute f^n+1
- back out tend as f^n+1 - f^n

In this context, the fact that the tendency represents a "future" change is somewhat arbitrary.